### PR TITLE
Implements functional api call to that fetches documents from db

### PR DESCRIPTION
**Pull request by DavidFuent1**
Implements API call for deliverable. 
The changes are split about 50/50 in the database/index.js and server/server.js

In database/index.js:
changes begin on line 19. Implements getAllPhotos function that runs a query to the server with a limit of 7 and passes the results to a callback.

in server/server.js: 
Changes line 8 to import the db file, but the bulk of the changes start on line 19. Refactors existing app.get() to call db.getAllPhotos() function. Defines callback to res.send() the results of the query.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -16,11 +16,20 @@ let photoSchema = mongoose.Schema({
...
+};
 
 
 
 module.exports = {
   Photos,
-  db
+  db,
+  getAllPhotos
 }
```
Comment (ingdjason) [inline database/index.js]: where u define db that u use in line 33, why you export Photos if you export getAllphotos that's connected

File: server/server.js [hunk 2]
```diff
@@ -14,7 +17,9 @@ app.use(express.urlencoded({ extended: true }));
 app.use('/', express.static('client/public'));
 
 app.get('/api/photos', (req, res) => {
-  res.send('Hello World');
+  db.getAllPhotos((err, result) => {
+    res.send(result);
+  });
```
Comment (ingdjason) [inline server/server.js]: In what format do you whant to send the data in your request ? 
take a look on that -> 
res.send uses content-type:text/html
res.json uses content-type:application/json

File: database/index.js (hunk 1 above)
Comment (DavidFuent1) [reply database/index.js]: Hey @ingdjason I'm exporting the model to seed.js separately to be used as a constructor for my seeding data!

File: server/server.js (hunk 2 above)
Comment (DavidFuent1) [reply server/server.js]: @ingdjason Good catch! Refactored to res.json()

**Pull request merged**